src/Chapter1.hs

```diff
@@ -631,8 +642,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integral a => a -> a
+sumLast2 n = 
+    let last2Digits = n `mod` 100
+        first = last2Digits `div` 10
+        second = last2Digits `mod` 10
+    in first + second
```

Comment:
Just a note, `first` and `second` are the common functions in the standard library, so it is usually better to use other name for your local vars 👌🏼